bpf/kmesh/cgroup_sock.c

```diff
@@ -65,12 +75,54 @@ static inline bool check_kmesh_enabled(struct bpf_sock_addr *ctx)
     return bpf_map_lookup_elem(&map_of_manager, &cookie);
 }
 
+static inline bool check_bypass_enabled(struct bpf_sock_addr *ctx)
+{
+    __u64 cookie = bpf_get_netns_cookie(ctx);
+    return bpf_map_lookup_elem(&map_of_bypass, &cookie);
+}
+
+static inline void record_bypass_netns_cookie(struct bpf_sock_addr *ctx)
+{
+    int err;
+    int value = 0;
+    __u64 cookie = bpf_get_netns_cookie(ctx);
+    err = bpf_map_update_elem(&map_of_bypass, &cookie, &value, BPF_NOEXIST);
+    if (err)
+        BPF_LOG(ERR, KMESH, "record netcookie failed!, err is %d\n", err);
+}
+
+static inline void remove_bypass_netns_cookie(struct bpf_sock_addr *ctx)
+{
+    int err;
+    __u64 cookie = bpf_get_netns_cookie(ctx);
+    err = bpf_map_delete_elem(&map_of_bypass, &cookie);
+    if (err && err != -ENOENT)
+        BPF_LOG(ERR, KMESH, "remove netcookie failed!, err is %d\n", err);
+}
+
+static inline bool conn_from_bypass_sim_add(struct bpf_sock_addr *ctx)
+{
+    // daemon sim connect 0.0.0.0:931(0x3a3)
+    // 0x3a3 is the specific port handled by the daemon for enable Kmesh
```

Comment:
Annotation Error.
It's not to mark enable kmesh but enable bypass.

Reply:
> Annotation Error. It's not to mark enable kmesh but enable bypass.

I've changed it.